csi/moac/nexus.ts

```diff
@@ -165,6 +166,11 @@ export class Nexus {
     this._emitMod();
   }
 
+  // Return true if the nexus is reachable (online).
+  isOffline() {
+    return !(this.node && this.node.isSynced());
+  }
```

Comment:
The comment appears to state the opposite of the function. Doesn't the function return true if the nexus is offline?

Reply:
You are absolutely right 🤦  let me fix that 👍